### PR TITLE
Fix: Undefined dom Element error fixed on page load

### PR DESCRIPTION
**Pull request by sulaimon23**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [] My code follows the code style of this project.
- [] This PR does not contain plagiarized content.
- [] The title and description of the PR is clear and explains the approach.
- [] I am making a pull request against the **dev branch** (left side).
- [] My commit messages styles matches our requested structure.
- [] My code additions will fail neither code linting checks nor unit test.
- [] I am only making changes to files I was requested to.

# Screenshots/ Videos

### CODE REVIEW

**Comment by billmal071** [comment]
@ickynavigator

File: packages/root-config/src/zuri-root-config.js [hunk 1]
```diff
@@ -76,6 +76,6 @@ window.addEventListener("zuri-plugin-load", () => {
 function removeLoader() {
   setTimeout(() => {
     const element = document.getElementById("single-spa-loader");
-    element.remove();
+    element?.remove();
```
Comment (ickynavigator) [inline packages/root-config/src/zuri-root-config.js:79]: is the element not always available?

File: packages/ui/src/message-board/MessageBoard.jsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ function MessageBoard({
   return (
     <>
       <MessageBoardContainer height={height}>
-        <div className="MsgBoard" onScroll={onHandleScroll}>
+        <div className="MsgBoard" onScroll={e => onHandleScroll(e)}>
```
Comment (ickynavigator) [inline packages/ui/src/message-board/MessageBoard.jsx:96]: this shouldn't be necessary

**Review by ickynavigator — CHANGES_REQUESTED** [review]

File: packages/root-config/src/zuri-root-config.js (hunk 1 above)
Comment (sulaimon23) [reply packages/root-config/src/zuri-root-config.js:79]: Yes

File: packages/ui/src/message-board/MessageBoard.jsx (hunk 2 above)
Comment (sulaimon23) [reply packages/ui/src/message-board/MessageBoard.jsx]: Fixed

**Comment by sulaimon23** [comment]
@JulianaSau

**Review by ickynavigator — APPROVED** [review]

**Pull request merged**